addons/launcher-template/launcher-template.addon

```diff
@@ -0,0 +1,4 @@
+# Name: launcher-template
+# Description: Imports the launch.openshift.io template
+
+oc create -f https://raw.githubusercontent.com/openshiftio/launchpad-templates/master/openshift/launchpad-template.yaml -n openshift
```

Comment:
@gastaldi  I somehow not comfortable with pulling the template for the runtime from https://github.com/openshiftio/launchpad-templates/tree/master/openshift . I would be ok for with pulling it from internet if it is pinned to a particular version (which is not possible with add-ons). The issue is how to make sure `https://raw.githubusercontent.com/openshiftio/launchpad-templates/master/openshift/launchpad-template.yaml` is a valid and correct URL always. Also how to make sure  launchpad-template.yaml always stays sane.